modules/client-handler/src/main/java/org/apache/ignite/client/handler/JdbcQueryEventHandlerImpl.java

```diff
@@ -299,16 +299,16 @@ private QuerySingleResult createJdbcResult(SqlCursor<List<?>> cur, QueryExecuteR
             case EXPLAIN:
             case QUERY:
                 return new QuerySingleResult(cursorId, fetch, !hasNext);
-            case DML:
-            case DDL: {
+            case DML: {
                 if (!validateDmlResult(fetch, hasNext)) {
                     return new QuerySingleResult(Response.STATUS_FAILED,
                             "Unexpected result for DML query [" + req.sqlQuery() + "].");
                 }
 
                 return new QuerySingleResult(cursorId, (Long) fetch.get(0).get(0));
-            }
-            default:
+            } case DDL: {
```

Comment:
what reason to have braces for DML and DDL cases?

Reply:
fixed